client/cypress/e2e/home-spec.cy.js

```diff
@@ -1,26 +1,29 @@
 describe('Home page', () => {
   beforeEach(() => {
+    cy.intercept('GET', 'http://localhost:3001/api/v1/entries', {
+      statusCode: 200,
+      fixture: 'entries.json'
+    })
     cy.visit('http://localhost:3000/')
   })
 
-  it('Should display the home page', () => {
+  it('Should onlg display the home page', () => {
     cy.get('.home').should('be.visible')
+    cy.get('header').should('not.exist')
   })
 
   it('Should display the site logo', () => {
     cy.get('.logo-home').should('be.visible').contains('BEAM')
   })
 
-  it('Should display a heading for the site info blurb' , () => {
-    cy.get('.home > :nth-child(2)').should('be.visible').contains('Beam is an interactive self-care feeling tracking app.')
+  it('Should display a dropdown for the site info blurb' , () => {
+    cy.get('.whats-beam').should('be.visible').contains('What is Beam?')
```

Comment:
i love how you use the` be.visible`, i hadn't used that before! love how simple it makes this